more-mime.types

```diff
@@ -0,0 +1,51 @@
+# Additional extensions
+
+text/x-R			r R Rhistory Rout Rout.save Rout.fail
+text/x-ada		adb ads
+text/x-asp		asp
+text/x-bibtex	bib
+text/x-c			c
+text/x-chdr		h hh hp hpp h++
+text/x-cpp		cpp c++ cxxcc C
+text/x-csharp		cs
+text/css				css CSSL
+text/x-dsrc			d
+text/x-diff			diff path
+text/vnd.graphviz		dot gv
+text/x-dtd			dtd
+text/x-eiffel		e eif
+text/x-erlang		erl erh
+text/x-forth		frt fs
+text/x-fortran	f f90 f95 for
+text/x-fsharp		fs
+text/x-gap			g gd gi gap
+text/x-gtkrc		gtkrc
+text/x-literate-haskell		lhs
+text/x-haskell						hs
+text/x-idl				idl
+text/x-ini-file		ini
+text/x-javascript		js
```

Comment:
I removed it. Anyway, it was not in the final mime_types.ml file, as js is already associated in mime.types.